drv/gimlet-seq-server/src/main.rs

```diff
@@ -134,7 +134,6 @@ fn main() -> ! {
             // Tell everyone that something's broken, as loudly as possible.
             ringbuf_entry!(Trace::StartFailed(SeqError::I2cFault));
             sys.gpio_set(FAULT_PIN_L);
-            jefe.set_state(PowerState::InitFailed as u32);
```

Comment:
Note that with this change, you could pass jefe into the init function by-value. (I think this is the only use after the call to init.)